digitaltwin_client/readme.md

```diff
@@ -2,34 +2,35 @@
 
 This folder contains the headers, source, samples, and internal testing for the Digital Twin Client for the C SDK.
 
-These instructions and samples assume basic familiarity with Digital Twin concepts.  To learn more background information, see [here](https://aka.ms/iotpnpdocs).
+These instructions and samples assume basic familiarity with Digital Twin and Azure IoT Plug and Play concepts.  To learn more, please go [here](https://aka.ms/iotpnpdocs).
 
 ## Getting started
 ### Building
-Instructions on how to build the Digital Twin Client SDK or else retrieve packages for it are available [here](./doc/building_sdk.md).  If you are familiar with building the IoTHub Device SDK, the instructions are almost identical.
+Start by building the Digital Twin Client SDK -- instructions are available [here](./doc/building_sdk.md).  Please use these instructions as they include cmake flags specific to the samples.  On Windows, be sure to clone the repo near your system root to maintain a valid path length.
```

Comment:
cmake incaps -> CMAKE

"your system root" this implies your system drive aka. c:\  -> your drives root folder

Reply:
Can you please clarify CMAKE or CMake is preferred?  https://cmake.org/ shows CMake.

Yes.  The system root is what I meant based on recommendation.  I could not build when I put the repo under my user directory because the path was too long.

Reply:
CMake 

The system root is the c:\ drive. Will you have issues if you put it on d:\ which is not the system root? I assume you just want it on the root of the drive and not the "system" root

Reply:
I see what you mean.  Yes the root, not necessarily the system root. Thanks.